### ISSUE TITLE
About the semi-surpervised method which is hard to realize in keras

### ISSUE THREAD

**Issue by SUNSONG111** [keras-team/keras#14415]
I want to make a network in which there are two branches in Keras, these two branches shared the same weight value all the time. The first branch is a supervised branch, it needs input data and labels. The second branch just used for feature extraction.
In fact, I have tried for a lot of time, and finally, I conducted it on pytorch.
I just want to know, is it a mishap of Keras which is not possible to solve?
This is the structure I really want to retrieve, can anybody help me?
![image](https://user-images.githubusercontent.com/20466308/106387063-edfe7f80-6412-11eb-879f-5a64b55417b4.png)

**Comment by sushreebarsa** [comment]
@SUNSONG111 In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by innat** [comment]
If you can ask questions regarding this on stack-overflow and link the question here, I would be glad to look into it. (I think this is fit on there).

**Comment by sushreebarsa** [comment]
This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you!
